orangecontrib/text/preprocess/normalize.py

```diff
@@ -1,3 +1,4 @@
+from functools import lru_cache
```

Comment:
Is this ever used?

Reply:
I forgot to remove it after I change my implementation. It is now removed. Thanks